### PR TITLE
Added clarification of text for Language translations

### PR DESCRIPTION
**Pull request by Ambertvu**
## Description

I've clarified the introduction of the language file page. I noticed that some had a tough time finding the correct syntax and explanation on how to get and create the files. I hope that by first giving a short explanation and clearly describing the `key` and `area` elements it will be easier to pickup.

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [ ] New content
* [ ] Updates related to a new version
* [x] Other

### CODE REVIEW

**Review by alina-tincas — CHANGES_REQUESTED** [review]
Hi @Ambertvu, thank you for the PR 🙌 

An example with clarification is indeed nice to have! I will just run this over by our developers to confirm the example, and meanwhile I have made a suggestions to replace the community link with https://community.umbraco.com/. Let me know what you think about the suggestion 😊

**Comment by Ambertvu** [comment]
Done and done, thanks!

**Review by bergmania — APPROVED** [review]

File: 10/umbraco-cms/extending/language-files.md [hunk 1]
```diff
@@ -9,11 +9,31 @@ description: >-
 
 # Language Files & Localization
 
-Language files are used to translate:
+Language files are XML files used to translate:
 - The Umbraco backoffice user interface so that end users can use Umbraco in their native language. This is particularly important for content editors who do not speak English.
 - The member identity errors in an Umbraco website enabling end users to use Umbraco in the website language.
+- Add translations for your packages [see here for docs on how to include translations for your own package](packages/language-files-for-packages.md)
```
Comment (alina-tincas) [inline 10/umbraco-cms/extending/language-files.md:15]: To make the link text shorter and hence readable that it is a link:
```suggestion
-  Read [Add translations for your packages](packages/language-files-for-packages.md) article to see how to include translations for your own package.
```

File: 11/umbraco-cms/extending/language-files.md [hunk 2]
```diff
@@ -6,11 +6,31 @@ description: >-
 
 # Language Files & Localization
 
-Language files are used to translate:
+Language files are XML files used to translate:
 - The Umbraco backoffice user interface so that end users can use Umbraco in their native language. This is particularly important for content editors who do not speak English.
 - The member identity errors in an Umbraco website enabling end users to use Umbraco in the website language.
+- Add translations for your packages [see here for docs on how to include translations for your own package](packages/language-files-for-packages.md)
```
Comment (alina-tincas) [inline 11/umbraco-cms/extending/language-files.md:12]:
```suggestion
-  Read [Add translations for your packages](packages/language-files-for-packages.md) article to see how to include translations for your own package.
```

File: 12/umbraco-cms/extending/language-files.md [hunk 3]
```diff
@@ -6,11 +6,31 @@ description: >-
 
 # Language Files & Localization
 
-Language files are used to translate:
+Language files are XML files used to translate:
 - The Umbraco backoffice user interface so that end users can use Umbraco in their native language. This is particularly important for content editors who do not speak English.
 - The member identity errors in an Umbraco website enabling end users to use Umbraco in the website language.
+- Add translations for your packages [see here for docs on how to include translations for your own package](packages/language-files-for-packages.md)
```
Comment (alina-tincas) [inline 12/umbraco-cms/extending/language-files.md:12]:
```suggestion
-  Read [Add translations for your packages](packages/language-files-for-packages.md) article to see how to include translations for your own package.
```

**Review by alina-tincas — CHANGES_REQUESTED** [review]
Hi @Ambertvu thank you for your patience! The code examples look good.

I have just one suggestion, can you take a look at it and let me know what you think? 😊

**Comment by Ambertvu** [comment]
@alina-tincas I did a small change to your suggestion (not sure how to do it on github, so I just approved and changed them)

I removed the 'article' word.

I think it's reads better with:
'Read the xxxx article to see how..`
or this felt even better:
'Read xxxx to see how..`

and not
`Read xxx article to see how..`

**Comment by alina-tincas** [comment]
That is all right @Ambertvu 😊 

Merging it in, thank you again for the PR 🙌

**Comment by alina-tincas** [comment]
(as for making changes, you can do them locally on the same branch you have pushed those changes to, or you can just commit the suggestions directly in github)

**Comment by Ambertvu** [comment]
Anytime!

Yea that's what I did, hah, thought maybe there was an other way. Thanks!

**Pull request merged**